Detectors/TRD/base/include/TRDBase/EventDisplayFeed.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_TRD_EVENTDISPLAYFEED_H
+#define O2_TRD_EVENTDISPLAYFEED_H
+
+namespace o2
+{
+namespace trd
+{
+
+class EventDisplayFeed
```

Comment:
Yeah, I thought so too, but when I delete the file, O2 fails to compile because it looks for this file. Not sure why?

Reply:
That cannot be, since it is not even added to a CMakeLists.txt file. Whether the file is present or not does not make a difference

Reply:
`2021-08-19@10:31:09:DEBUG:O2:O2:0: Detectors/TRD/base/G__O2TRDBase.cxx:62:10: fatal error: /root/alice/sw/SOURCES/O2/event-display-workflow/0/Detectors/TRD/base/include/TRDBase/EventDisplayFeed.h: No such file or directory
2021-08-19@10:31:09:DEBUG:O2:O2:0:    62 | #include "/root/alice/sw/SOURCES/O2/event-display-workflow/0/Detectors/TRD/base/include/TRDBase/EventDisplayFeed.h"
2021-08-19@10:31:09:DEBUG:O2:O2:0:       |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-08-19@10:31:09:DEBUG:O2:O2:0: compilation terminated.`

I don't know how to explain it either...

The only possible connection I see is that the event display workflow is linked to `O2::TRDWorkflow` which is linked to `O2::TRDBase`. Maybe it expects this header file because of this, but I'm just guessing.

Reply:
But since EventDisplayFeed is not part of `O2::TRDBase` (have a look at Detectors/TRD/base/CMakeLists.txt where the library is defined) it is not needed at all.
I suspect that your build environment is somehow not cleaned up. You can try to remove everything from your O2 build directory and build again. Unfortunately this does recompile O2 completely of course :/
I bet that if you remove that file and push it to github O2 will compile here just fine ;)
But you need to fix the erros of the fullCI:
`/sw/SOURCES/O2/6865-slc8_x86-64/0/Detectors/TRD/workflow/include/TRDWorkflow/TRDEventDisplayFeedSpec.h:34:3: error: use '= default' to define a trivial default constructor [modernize-use-equals-default]`